mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
```

Comment:
The line 'res.json({ message: "Order deleted successfully" });' is duplicated in the code. This duplication can be removed to make the code cleaner and more efficient. 

Additionally, it would be better to define a constant for the success message "Order deleted successfully" and use it throughout the code to maintain consistency. This would also make it easier to change the message in the future if needed.